src/features/content-insights/ContentAnalyticsErrorState.tsx

```diff
@@ -1,18 +1,59 @@
 import * as React from 'react';
 import { FormattedMessage } from 'react-intl';
 
-import HatWand140 from '../../illustration/HatWand140';
+import PropTypes from 'prop-types';
+import MetricsReview56 from '../../illustration/MetricsReview56';
 import messages from './messages';
+import { ResponseError } from './types';
 
 import './ContentAnalyticsErrorState.scss';
 
-const ContentAnalyticsErrorState = () => (
-    <div className="ContentAnalyticsErrorState" data-testid="ContentAnalyticsErrorState">
-        <HatWand140 data-testid="ContentAnalyticsErrorState-image" />
-        <div className="ContentAnalyticsErrorState-text">
-            <FormattedMessage {...messages.contentAnalyticsErrorText} />
+const SIZE_MAP = {
+    small: 56,
+    medium: 80,
+    large: 95,
+};
+interface Props {
+    error: ResponseError;
+    size?: 'small' | 'medium' | 'large';
+}
+
+const ContentAnalyticsErrorState = ({ error, size = 'small' }: Props) => {
+    const renderErrorMessage = () => {
+        const isPermissionError = error.status === 403;
+
+        if (isPermissionError) {
+            return (
+                <div
+                    className="ContentAnalyticsPermissionError-text"
```

Comment:
I think you would update the descendant or use a modifier in this case:

```suggestion
                    className="ContentAnalyticsErrorState-permissionText"
```
```suggestion
                    className="ContentAnalyticsErrorState-text--permission"
```